spec/models/veteran_onboarding_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VeteranOnboarding, type: :model do
```

Comment:
I already approved so 🤷 but I suppose the Flipper logic should be in this spec now since it was moved into the VeteranOnboarding class